### ISSUE TITLE
Any updates about V3? is this the end of 3.5 - 3.9 WebOS systems ☹️

### ISSUE THREAD

**Issue by trxshnumb** [RootMyTV/RootMyTV.github.io#111]

**Comment by throwaway96** [comment]
[GetMeIn](https://forum.xda-developers.com/t/getmein-one-time-rooting-jailbreaking-tool-for-webos-lg-tvs.3887904/) might work on webOS 3.x. You'll probably have to do the steps regarding `jail_app.conf` from the [crashd guide](https://gist.github.com/throwaway96/e811b0f7cc2a705a5a476a8dfa45e09f) to get access to `/dev/mem` from the dev mode jail. There's also a [hardware method](https://gist.github.com/throwaway96/827ff726981cc2cbc46a22a2ad7337a1).

**Comment by trxshnumb** [comment]
how do i do that jail_app.conf step with getmein method
im on 3.9.2 webos btw 😭

**Comment by throwaway96** [comment]
Like I said, follow the guide. Steps 1-7.

**Comment by trxshnumb** [comment]
okay after crashd step 7, i should get on getmein method right?

**Comment by throwaway96** [comment]
Yes. There are problems with GetMeIn's built-in payload (it runs a bunch of commands you can't change without hacking a different embedded binary into it), but it still pops a root shell.
